HAB/HAB.ino

```diff
@@ -39,7 +39,7 @@ void setup() {
 
 void loop() {
   GPS::GPSData gpsData = GPS::process();
-  if (gpsData.isValid) {
+  
```

Comment:
Can you fix the indentation on the following lines? They were indented when there was the `if` statement, but that was removed.

Reply:
Done!